src/UI/PageState/types.ts

```diff
@@ -1,5 +1,5 @@
 export interface PageStateProps {
-  title?: string;
+  title?: React.ReactNode;
```

Comment:
Maybe we should allows both string and React node?

Reply:
String is a valid React.ReactNode, so both work